crates/aptos/src/genesis/keys.rs

```diff
@@ -28,6 +28,8 @@ const VALIDATOR_FILE: &str = "validator-identity.yaml";
 const VFN_FILE: &str = "validator-full-node-identity.yaml";
 
 /// Generate account key, consensus key, and network key for a validator
+///
+/// These keys are used for running a validator or operator in a network
```

Comment:
I am going to write a short doc where I will say "validator" refers to the machine and "operator" refers to the persona. This is indeed a nit, but if you remove "or operator" that would be better.